x/staking/app_test.go

```diff
@@ -65,7 +65,7 @@ func TestStakingMsgs(t *testing.T) {
 	// create validator
 	description := types.NewDescription("foo_moniker", "", "", "", "")
 	createValidatorMsg, err := types.NewMsgCreateValidator(
-		sdk.ValAddress(addr1), valKey.PubKey(), bondCoin, description, commissionRates, sdk.OneInt(),
+		sdk.ValAddress(addr1), valKey.PubKey(), bondCoin, description, commissionRates,
```

Comment:
Will we need to do an upgrade handler that prunes all Validators's self delegation attributes?

Reply:
yeah we'll have to migrate all changed protos